src/components/billing-chart/billing-chart.tsx

```diff
@@ -52,6 +45,100 @@ interface BillingChartProps {
   diagrammType: "Bar" | "Line";
 }
 
+const TooltipContainer = styled.div`
+  border-radius: 4px;
+  border: none;
+  background: white;
+  padding: 16px;
+`;
+
+const TooltipHeading = styled.h4`
+  font-style: normal;
+  font-weight: 900;
+  font-size: 12px;
+  text-transform: uppercase;
+  margin: 0;
+  padding: 0;
+  color: #202840;
+`;
+
+const TooltipText = styled.p`
+  font-size: 14px;
+  color: #202840;
+`;
+
+const TooltipValue = styled.p`
+  font-weight: bold;
+  font-size: 14px;
+  color: #202840;
+  margin-left: 12px;
+`;
+
+const TooltipRow = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 8px;
+`;
+
+const TooltipDot = styled.span<{ repositoryColor?: string }>`
+  ${(props: { repositoryColor?: string }) => `color: ${props.repositoryColor}`};
+  font-size: 8px;
+  vertical-align: 2px;
+  margin-right: 8px;
+`;
+
+const CustomTooltip = ({
+  active,
+  payload,
+  label,
+}: TooltipProps<number, string>) => {
+  if (
+    !payload ||
+    payload.length === 0 ||
+    payload.every((tooltipEntry) => tooltipEntry.value === 0)
+  ) {
+    return (
+      <TooltipContainer>
+        <TooltipHeading>{tooltipLabelFormatter(label)}</TooltipHeading>
+        <TooltipText>no expenses, yeah 🎉</TooltipText>
+      </TooltipContainer>
+    );
+  }
+
+  if (active && payload && payload.length) {
+    payload.sort((a: Payload<number, string>, b: Payload<number, string>) => {
+      if (a.value && b.value) {
```

Comment:
You can put this into a function and give it a meaningful name, like `sortByPrice`